launchpod/angular-launchpod/backend/src/test/java/com/google/launchpod/TranslationServletTest.java

```diff
@@ -0,0 +1,180 @@
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.Translate.TranslateOption;
+import com.google.cloud.translate.Translation;
+import com.google.cloud.translate.testing.RemoteTranslateHelper;
+import com.google.launchpod.data.Item;
+import com.google.launchpod.data.RSS;
+import com.google.launchpod.servlets.TranslationServlet;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+/**
+ * Runs unit tests for the FormHandlerServlet that contains doPost(), doGet(),
+ * and xmlString() methods.
+ */
+@RunWith(JUnit4.class)
+public class TranslationServletTest extends Mockito {
+
+  @InjectMocks
+  private TranslationServlet servlet = new TranslationServlet();
+
+  @Mock
+  HttpServletRequest request;
+
+  @Mock
+  HttpServletResponse response;
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  private final LocalServiceTestHelper helper = new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  // keys
+  private static final String USER_FEED = "UserFeed";
+  private static final String LANGUAGE = "language";
+  private static final String RSS_FEED_LINK = "rssFeedLink";
+  private static final String TEST_PODCAST_TITLE = "American Life";
+  private static final String TEST_TARGET_LANGUAGE = "es";
+  private static final String TEST_LANGUAGE = "en";
+  private static final String TEST_DESCRIPTION = "Episode generated by launchpod";
+  private static final String TEST_MP3_LINK = "this-is-a-test.mp3";
+  private static final String TEST_ID = "123456";
+  private static final String TEST_EMAIL = "123@abc.com";
+  private static final String BASE_URL = "https://launchpod-step18-2020.appspot.com/rss-feed?id=";
+  private static final RSS TEST_RSS_FEED = new RSS(TEST_PODCAST_TITLE, TEST_DESCRIPTION, TEST_LANGUAGE, TEST_EMAIL,
+      TEST_MP3_LINK);
+  private RemoteTranslateHelper translateHelper = RemoteTranslateHelper
+      .create("AIzaSyBlu9s7xFLjlHAdvlVuUISq_MbyELuCRZo");
+  private Translate translateMock = translateHelper.getOptions().getService();
+
+  @Before
+  public void setUp() {
+    MockitoAnnotations.initMocks(this);
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  /**
+   * Test translate helper to ensure it translates the given string
+   */
+  @Test
+  public void doPost_TranslateHelperWorks() {
```

Comment:
"Works" is ambiguous - specify exactly what you mean by works

in this case, i believe you mean that the description is translated